Makefile

```diff
@@ -29,6 +36,12 @@ test:
 	docker-compose run --rm web rake
 
 deploy:
-	docker build -t carwashapp:0.0.5 .
-	docker tag carwashapp:0.0.5 mmmikem/car-wash-app:0.0.5
-	docker push mmmikem/car-wash-app:0.0.5
+	docker build --target staging -t carwashapp:staging .
+	docker tag carwashapp:staging mmmikem/car-wash-app:staging
+	docker push mmmikem/car-wash-app:staging
+
+image = carwashapp:0.0.6
```

Comment:
Would something like `image = cat .version`  work here? The version seems to get used in a place or two, so might be nice to have a file where you can just change it in one single place. :)